pyilc/wavelets.py

```diff
@@ -114,10 +114,31 @@ def CosineNeedlets(self,ellmin=None, ellpeaks=None, ): # Fiona added CosineNeedl
         return self.ell, self.filters
 
     # scale-discretized wavelets
-    #def ScaleDiscretizedWavelets(self, TODO):
-    #    TODO: (not yet implemented here)
-    #    # simple check to ensure that sum of squared transmission is unity as needed for NILC algorithm
-    #    assert (np.absolute( np.sum( self.filters**2., axis=0 ) - np.ones(self.ELLMAX+1,dtype=float)) < self.tol).all(), "wavelet filter transmission check failed"
+    def ScaleDiscretizedWavelets(self,ellboundaries =None):
```

Comment:
Can you add a reference for these?  I'm not sure they match exactly what I had in mind, which were the wavelets from Sec. 2 of https://arxiv.org/abs/1211.1680